src/services/query.ts

```diff
@@ -30,6 +31,7 @@ export const buildResourceQuery = (
     ...defaultQueryParams,
     resource,
     ...queryParamsOverrides,
+    datasource: { uid: VAR_DATASOURCE_EXPR },
```

Comment:
I think we would need the same in `buildDataQuery`, right?

Reply:
Good question, I think since it always redirects back to the service selection scene whenever the datasource is changed that we won't see the impact yet?

Although when I make this change locally I do see some queries that get run after the datasource change, but before the navigation that get cancelled immediately. 

Let's not change it in this PR, and I'll keep this in mind as I continue the refactoring into the runtime datasource.